Content/GUI/GrimoireSelection/GrimoireInvButton.cs

```diff
@@ -0,0 +1,68 @@
+﻿using PathOfTerraria.Content.GUI.GrimoireSelection;
+using PathOfTerraria.Content.Items.Gear.Weapons.Grimoire;
+using PathOfTerraria.Core.Loaders.UILoading;
+using System.Collections.Generic;
+using Terraria.Audio;
+using Terraria.ID;
+using Terraria.UI;
+
+namespace PathOfTerraria.Content.GUI.PlayerStats;
+
+public class GrimoireInvButton : SmartUIState
```

Comment:
I think this needs to get added to `UiStateRefresher` as a temp solve for 2k/4k monitors